service/controller/v12/resource/instance/create.go

```diff
@@ -209,8 +214,8 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		// worker instance selected to be reimaged in order to not update its
 		// version information. The next reconciliation loop will catch up here
 		// and instruct the worker instance to be reimaged again.
-		var workerUpgraded bool
-		if !masterUpgraded {
+		var workerUpgradeInProgess bool
+		if !masterUpgradeInProgress {
```

Comment:
`masterUpgraded` and `masterUpgradeInProgress` are semantically different. Yet the condition negates with `!` in both cases. That looks wrong to me. How can this work that way now? Was the initial wording just wrong?

Reply:
Yes, as far as I understood the original wording was misleading, thus the rename.